pkgs/tools/text/ripgrep/default.nix

```diff
@@ -1,23 +1,28 @@
-{ stdenv, fetchFromGitHub, rustPlatform, makeWrapper }:
+{ stdenv, fetchFromGitHub, rustPlatform, makeWrapper, asciidoc, libxslt }:
 
-with rustPlatform;
-
-buildRustPackage rec {
+rustPlatform.buildRustPackage rec {
   name = "ripgrep-${version}";
-  version = "0.7.1";
+  version = "0.8.0";
 
   src = fetchFromGitHub {
     owner = "BurntSushi";
     repo = "ripgrep";
     rev = "${version}";
-    sha256 = "0z3f83vhy464k93bc55i9lr6z41163q96if938p9ndhx2q3a20ql";
+    sha256 = "0ha46vhma2diqxfgasdc9gwlkcrlhc4l65nayvdy4m2ah5bm4qp6";
   };
 
-  cargoSha256 = "1d6s01gmyfzb0vdf7flq6nvlapwcgbj0mzcprzyg4nj5gjkvznrn";
+  cargoSha256 = "0q44qa9myrzg42q0lvclpk5nypmf0q7v3xy5nnsb28j3imvcrs4p";
+
+  nativeBuildInputs = [ asciidoc libxslt ];
 
   preFixup = ''
     mkdir -p "$out/man/man1"
-    cp "$src/doc/rg.1" "$out/man/man1"
+    cp target/release/build/ripgrep-*/out/rg.1 "$out/man/man1/"
```

Comment:
I build with your changes locally and the manpage can be found in the following path: `./source/target/release/build/ripgrep-f969427e20cd23eb/out/rg.1.txt`

Reply:
Manpage should be from this file with `a2x --no-xmllint --doctype manpage --format manpage rg.1.txt` (see https://github.com/BurntSushi/ripgrep/blob/master/build.rs#L73:L97).

Reply:
Ahh yeah :( Any idea why the correct file is not created?

Reply:
Unable to reproduce on NixOS and other Linux distribution

Reply:
Hmm, for me it still does not work. If you want, I can look at it tomorrow.

Reply:
I can reproduce the error

Reply:
@bkchr it would be great

Reply:
I messed up. Sandboxing was actually disabled, so I bet it is it -_-

Reply:
I got it to work with `asciidocFull` instead of `asciidoc` (#34967). `asciidoc` should also be added to `buildInputs`.

Reply:
Why can't it be a native input? It's a build time dependency.

Reply:
Yeah you're right, it should be a native input.